### PR TITLE
Updating minimum PHP version to 5.6

### PR DESCRIPTION
**Pull request by rawkode**
I am targetting this branch, because it's a PHP version change / potential BC break

Refs # https://github.com/sonata-project/dev-kit/issues/208
## Changelog

``` markdown
### Deprecated
- PHP versions 5.3 - 5.5
```

## Subject

Updating minimum PHP version to 5.6, as per https://github.com/sonata-project/dev-kit/issues/208

### CODE REVIEW

**Comment by rawkode** [comment]
This failure isn't something I can fix. Can you retrigger the check?

> We were unable to access the repo or commit to analyze it.
> It is possible that the commit was deleted from GitHub before we had time to analyze it.

**Comment by greg0ire** [comment]
I don't think we can, try rebasing maybe?

**Comment by rawkode** [comment]
@greg0ire Failed again. Perhaps something isn't configured correctly?

File: Tests/Model/PageTest.php [hunk 1]
```diff
@@ -13,11 +13,6 @@
 
 class PageTest extends \PHPUnit_Framework_TestCase
 {
-    /**
-     * NEXT_MAJOR: remove the legacy group from this test.
-     *
-     * @group legacy
-     */
     public function testSlugify()
```
Comment (core23) [inline Tests/Model/PageTest.php]: Why did you remove the legacy group and not the whole methode?

Comment (rawkode) [inline Tests/Model/PageTest.php]: Silly mistake. Removed

**Comment by greg0ire** [comment]
I have no idea why the styleCI build fails… maybe shoot them an email, there is an address on the StyleCI page.

**Comment by OskarStark** [comment]
@sonata-project/contributors is it a minor or a patch?

**Comment by greg0ire** [comment]
@OskarStark it's major
